FogCreek/Glitch-Community
# api-updates/1: Use v1 routes for create team pop

## Links
* Remix link: [https://hissing-gibbon.glitch.me/](https://hissing-gibbon.glitch.me/)
* Issue-tracker link: https://app.clubhouse.io/glitch/story/3159/port-teamid-byurl-url-and-userid-bylogin-login-to-v1-routes

## Changes:
* Changes two routes to use new V1 routes

## How To Test:
* Try to create a team, enter a known username and team name that are taken, you should see an error that says the name is unavailable
* An unused username and teamname should be allowed when creating a team
